app/(pages)/events/page.tsx

```diff
@@ -1,10 +1,43 @@
-import { Stack } from '@mui/joy';
+import EventsCard from '../../../components/EventsCard';
+
+import { Stack, Grid, Box } from '@mui/joy';
 import PageHeaderCard from "@/components/PageHeaderCard";
+import { Typography } from '@mui/joy';
 
 export default function Events() {
   return (
     <Stack>
       <PageHeaderCard imagePath={'/events/header.jpg'} pageTitle={'Events'}/>
+      <Typography level="h1" sx={{ mt: 10 }}>Upcoming Events</Typography>
+      <Typography level="body-md" sx={{ textAlign: 'center', mb: 10 }}>
+        WIESoc hold many events throughout the year, with a great mix of industry 
+        and social events. Come and join us for our upcoming events!
+      </Typography>
+      <Box
+        display="flex"
+        justifyContent="center"
+        sx={{ width: '100%', mb: 10 }}
+      >
+        <Grid 
+          container 
+          justifyContent="center"
+          sx={{ width: '70%' }}
+          spacing={5}
+        >
+          {Array.from(Array(6)).map((_, index) => (
+            <Grid 
```

Comment:
Might just be me but i'm getting this error here:
<img width="362" alt="image" src="https://github.com/UNSW-WIESoc/website/assets/119286195/95be3477-a455-4129-b7de-bbc294018c06">
Warning: Received `true` for a non-boolean attribute `item`.